src/keys.rs

```diff
@@ -24,6 +24,8 @@ pub enum KeyPress {
     ControlRight,
     /// Ctrl-↑
     ControlUp,
+    /// Control Backspace
```

Comment:
Maybe we should warn that this key can/may not be used on unix.

Reply:
I wasn't sure if you wanted that comment on ControlBackspace only or ShiftBackspace and MetaBackspace, so I only put it where you called it out.